usaspending_api/common/tests/integration/test_spark_dataframe_diff.py

```diff
@@ -241,7 +241,7 @@ def test_diff_one_change_single_col_compared(spark: SparkSession):
 
     data_right = deepcopy(data_left)
     changed_row_id = data_right[0]["id"]
-    data_right[0]["numeric_val"] = data_right[0]["numeric_val"] * 2
+    data_right[0]["numeric_val"] = data_right[0]["numeric_val"] + 1
```

Comment:
This test fails if `randint` returns 0. (2 * 0 = 0)  It is not related to the other changes in the PR, but the tests failed randomly due to this issue, so I decided to fix it in this PR instead of a different PR.